TDDPracticeTests/TDDPracticeTests/RomanNumbers/RomanNumberConverterTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using FluentAssertions;
+
+namespace TDDPracticeTests.RomanNumbers
+{
+    public class RomanNumberConverterTest
+    {
+        [Theory]
+        [InlineData(1, "I")]
```

Comment:
becomes unreadable, can you use something other than inline?

Reply:
Changed Inlinedata to memberdata.